src/components/Ribbon/Ribbon.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import Ribbon from './Ribbon';
+
+const meta = {
+  title: 'Components/Ribbon',
+  component: Ribbon,
+  tags: ['autodocs']
+} satisfies Meta<typeof Ribbon>;
+
+export default meta;
+type Story = StoryObj<typeof Ribbon>;
+
+export const Playground: Story = {
+  render: (args) => <Ribbon {...args}>Ribbon</Ribbon>
+};
```

Comment:
Can we do `export const Playground: Story = {}` here? I know it looks silly, but avoiding the render method makes storybook pick up on the default values for args better

Reply:
Going with the following so that there's something in the box:
```
export const Playground: Story = {
  args: {
    children: 'Ribbon'
  }
};
```